### DISCUSSION TITLE
clash core删库了

### DISCUSSION THREAD

**Discussion by chenmozhijin** [vernesong/OpenClash#3593] [General]
![image](https://github.com/vernesong/OpenClash/assets/110812055/022de0d7-133d-48f9-a9f7-f599209fe693)
![image](https://github.com/vernesong/OpenClash/assets/110812055/4d8239fa-53d0-42d4-a0de-f0119d48cbcc)
https://github.com/Dreamacro/clash/

**Comment by WYC-2020** [comment]
正常可不是只有这一个跑了，还有好几个

**Reply by openips** [reply]
过几天会再出来不

**Comment by chenmozhijin** [comment]
> 正常可不是只有这一个跑了，还有好几个

确实，还有：
https://github.com/Fndroid/clash_for_windows_pkg
https://github.com/Kr328/ClashForAndroid
https://github.com/jerrykuku/luci-app-vssr
https://github.com/Fclash/Fclash
https://github.com/yichengchen/clashX
https://github.com/wanhebin/clash-for-linux

还有archived的：
https://github.com/MetaCubeX/Clash.Meta
https://github.com/EAimTY/tuic
https://github.com/immortalwrt/homebridger
https://github.com/MetaCubeX/ClashMetaForAndroid
https://github.com/zzzgydi/clash-verge

**Reply by aksjdakwhdijahskjdhkajbsf** [reply]
那现在怎么下载啊

**Reply by hadifarnoud** [reply]
Clash.Meta renamed itself to Mihomo
the repo is not obvious it is Clash Meta but it's the same thing
